lib/template-loader.php

```diff
@@ -107,6 +107,42 @@ function gutenberg_find_template( $template_file ) {
 	if ( $template_query->have_posts() ) {
 		$template_posts            = $template_query->get_posts();
 		$_wp_current_template_post = array_shift( $template_posts );
+
+		// Build map of template slugs to their priority in the current hierarchy.
+		$slug_priorities = new stdClass();
```

Comment:
As opposed to using an object, I think this should rather just be an associative array, which will behave the same but be more normal in PHP.

```suggestion
		$slug_priorities = array();
```